### PR TITLE
Modify languages.yaml to adjust and fix some Chinese translation

### PR DESCRIPTION
**Pull request by leonardyan**
Long time no see! Also, in the settings/preferences panel, would you like to change the "Chinese" in language selection into "Chinese (simplified)"? Since I think there might be a traditional Chinese translation one day ;)

### CODE REVIEW

**Comment by leonardyan** [comment]
well i'll edit settings.py to do this

File: fastflix/widgets/settings.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger("fastflix")
 language_list = sorted((k for k, v in Lang._data["name"].items() if v["pt2B"] and v["pt1"]), key=lambda x: x.lower())
 
-known_language_list = ["English", "Chinese", "Italian", "French", "Spanish", "German"]
+known_language_list = ["English", "中文（简体）|Chinese (simplified)", "Italian", "French", "Spanish", "German"]
```
Comment (leonardyan) [inline fastflix/widgets/settings.py:19]: You may also remove the "中文（简体）|" to leave only the English part if desired

Comment (leonardyan) [reply fastflix/widgets/settings.py:19]: Or will this language list be localized with your l10n framework one day? If so please remove "中文（简体）|".

Comment (cdgriffith) [reply fastflix/widgets/settings.py:19]: Can't have this change for now, as it's programmed later to do the find between "Chinese" > "zho" in the yaml file.

But that is a good change to strive for, actually having languages in their own language would be beneficial.

Comment (leonardyan) [reply fastflix/widgets/settings.py:19]: Okay I'll try to revert this.

Comment (leonardyan) [reply fastflix/widgets/settings.py]: Reverted

**Comment by leonardyan** [comment]
Today I wrote a blogpost in Chinese to introduce FastFlix(I'm possibly the first to do so). Wish your work to be known by more users.

**Pull request merged**